nasa/earthdata-search
# EDSC-3463: Adds subscription editing

# Overview

### What is the feature?

Adds a modal to allow editing of subscription items

### What is the Solution?

Implement a modal

### What areas of the application does this impact?

Granule and Collection subscriptions lists

# Testing

### Reproduction steps

- **Environment for testing:** SIT
- **Collection to test with:** ANY

1. Add a subscription
2. Edit only the name
3. Confirm the name is changed

1. Edit a subscription
2. Change parameters
3. Confirm the parameters are changed

1. Confirm validation prevents editing of subscriptions with matching queries (the edit button is disabled)

# Checklist
